c2corg_ui/static/js/listswitchbutton.js

```diff
@@ -0,0 +1,48 @@
+goog.provide('app.ListSwitchButtonController');
```

Comment:
Indeed I didn't know how to optimize this redundancy. If so, maybe the service itself is useless then, and I could make a much simpler PR? What do you think in terms of modeling?